app/serializers/api/v1/connection_serializer.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class ConnectionSerializer < ActiveModel::Serializer
+      type :connections
+      attributes :id, :input_id, :output_id
```

Comment:
May be it makes sense instead just returning input_id and output_id as attributes, return those connected tasks as relationships?